frontend/src/index.css

```
@@ -53,61 +58,66 @@
 :root {
   --radius: 0.625rem;
 
+  /* Custom Warm/Blue Palette */
+  --background: #FFF4EA;
+  --foreground: #2D3748;
+  /* Dark slate for readability on cream */
```

Stylelint (17.2.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)